www/docs/server/validators.md

```diff
@@ -363,6 +363,31 @@ export const appRouter = t.router({
 export type AppRouter = typeof appRouter;
 ```
 
+### With [runtypes](https://github.com/pelotom/runtypes)
+
+```ts twoslash
+import * as t from 'runtypes';
+import { initTRPC } from '@trpc/server';
+
+export const t = initTRPC.create();
+
+const publicProcedure = t.procedure;
+
+export const appRouter = t.router({
+  hello: publicProcedure
+    .input(t.Record({ name: t.String }))
+    .output(t.Record({ greeting: t.String }))
+    .query(({ input }) => {
+      //      ^?
+      return {
+        greeting: `hello ${input.name}`,
+      };
+    }),
+});
+
+export type AppRouter = typeof appRouter;
```

Comment:
`t` & `t` :)

```suggestion
import { initTRPC } from '@trpc/server';
import * as T from 'runtypes';

const t = initTRPC.create();
const publicProcedure = t.procedure;

export const appRouter = t.router({
  hello: publicProcedure
    .input(T.Record({ name: T.String }))
    .output(T.Record({ greeting: T.String }))
    .query(({ input }) => {
      //      ^?
      return {
        greeting: `hello ${input.name}`,
      };
    }),
});

export type AppRouter = typeof appRouter;
```

You'll also need to add `runtypes` as a devDep to www for shiki to be able to compile.

I did it locally but couldn't push to your branch :/